app/scripts/directives/layout.dir.js

```diff
@@ -6,7 +6,7 @@ angular.module('yourStlCourts').directive('layout', function () {
 
     link: function ($scope, element) {
       $scope.$on('$stateChangeStart', function(event, toState){
-        if (toState.name === 'home' || toState.name === 'ticketSearch' || toState.name === 'about' || toState.name === 'help' || toState.name === 'info') {
+        if (toState.name === 'home' || toState.name === 'ticketSearch' || toState.name === 'about' || toState.name === 'help' || toState.name === 'info' || toState.name === 'error') {
```

Comment:
I looked up const and it is only part of ES2015.  Should I just use a var instead?